geoips_clavrx/plugins/modules/algorithms/absdiff_cth.py

```diff
@@ -0,0 +1,115 @@
+"""Absolute difference of CTH between two CLAVR-x files 1 timestep apart."""
+
+import logging
+
+import numpy as np
+import xarray as xr
+
+from geoips.data_manipulations.corrections import apply_data_range
+
+LOG = logging.getLogger(__name__)
+
+
+name = "absdiff_cth"
+interface = "algorithms"
+family = "xarray_dict_to_xarray"
+
+
+def call(
+    xarray_dict,
+    output_data_range=None,
+    input_units=None,
+    output_units=None,
+    min_outbounds="crop",
+    max_outbounds="crop",
+    norm=False,
+    inverse=False,
+    sun_zen_correction=False,
+    mask_night=False,
+    max_day_zen=None,
+    mask_day=False,
+    min_night_zen=None,
+    gamma_list=None,
+    scale_factor=None,
+):
+    """Apply data range and requested corrections to a single channel product.
```

Comment:
Update the docstring here. Make sure to mention that this requires 3D data with two times. Maybe also raise a warning if there are more than two times.

Reply:
Will do.